### DISCUSSION TITLE
Power displaying as "ON" rather than battery percentage

### DISCUSSION THREAD

**Discussion by h-bag** [GitSquared/edex-ui#1110] [Questions] [answered]
I've seen a lot of screenshots where the power statistic is showing a percentage, regarding the battery. However, for my laptop (Dell G3 3500 running Fedora 33), it just displays as "ON" as if it were a desktop machine or plugged in (it isn't).

Is there something I forgot to toggle, or is this a bug?

**Comment by GitSquared** [comment, marked as answer]
Sounds like a bug, what do you have in `/sys/class/power_supply`?

Edit: #1113

**Reply by h-bag** [reply]
> Sounds like a bug, what do you have in `/sys/class/power_supply`?

I got two directories there - **AC** and **BAT0**

**Reply by GitSquared** [reply]
Ok, what's the content of `/sys/class/power_supply/BAT0/uevent`?

**Reply by h-bag** [reply]
> Ok, what's the content of `/sys/class/power_supply/BAT0/uevent`?

```
POWER_SUPPLY_NAME=BAT0
POWER_SUPPLY_TYPE=Battery
POWER_SUPPLY_STATUS=Discharging
POWER_SUPPLY_PRESENT=1
POWER_SUPPLY_TECHNOLOGY=Li-poly
POWER_SUPPLY_CYCLE_COUNT=0
POWER_SUPPLY_VOLTAGE_MIN_DESIGN=11400000
POWER_SUPPLY_VOLTAGE_NOW=10611000
POWER_SUPPLY_CURRENT_NOW=3406000
POWER_SUPPLY_CHARGE_FULL_DESIGN=4474000
POWER_SUPPLY_CHARGE_FULL=4474000
POWER_SUPPLY_CHARGE_NOW=2317000
POWER_SUPPLY_CAPACITY=51
POWER_SUPPLY_CAPACITY_LEVEL=Normal
POWER_SUPPLY_MODEL_NAME=DELL PN1VN05
POWER_SUPPLY_MANUFACTURER=SMP
POWER_SUPPLY_SERIAL_NUMBER=1080
```

**Reply by GitSquared** [reply]
Yup, definitely a bug. Moving to #1113
